GooglePlacesAutocomplete.js

```diff
@@ -216,10 +218,19 @@ const GooglePlacesAutocomplete = React.createClass({
   },
 
   getCurrentLocation() {
-    let options = (Platform.OS === 'android') ? {enableHighAccuracy: true, timeout: 20000} : {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000};
+    let options = null;
+    if (this.props.enableHighAccuracyLocation)
+      options = (Platform.OS === 'android') ? {enableHighAccuracy: true, timeout: 20000} : {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000};
     navigator.geolocation.getCurrentPosition(
       (position) => {
-        this._requestNearby(position.coords.latitude, position.coords.longitude);
+        if (this.props.nearbyPlacesAPI === 'None') {
```

Comment:
Hi @bosquig 

Thanks for the explanation. Yes, you can add `None` option here.